backend/epub/epub-document.c

```diff
@@ -182,6 +182,9 @@ epub_document_check_hits(EvDocumentFind *document_find,
 	gchar *filepath = g_filename_from_uri((gchar*)page->backend_page,NULL,NULL);
 	htmlDocPtr htmldoc =  xmlParseFile(filepath);
```

Comment:
You can also check whether [xmlParseFile](http://xmlsoft.org/html/libxml-parser.html#xmlParseFile) returned NULL. In any case, I don't think it's an [application error (g_error)](https://developer.gnome.org/glib/stable/glib-Message-Logging.html#g-error) and maybe free the allocated memory that needs to be freed and just return 0.

Reply:
Ok, I would modify these as you said.